src/components/AppHeader/AppHeader.vue

```diff
@@ -86,4 +86,7 @@ export default {
 .app-header__tabs.v-tabs {
   width: auto;
 }
+.v-tabs-bar__content {
```

Comment:
There's this weird spacing to the right of the tabs that isn't there on the left side of the tabs, I think it would look nicer if that was fixed:

![Scherm­afbeelding 2023-05-04 om 14 21 29](https://user-images.githubusercontent.com/9058722/236202646-8bfcb2b0-2f23-44c3-93de-06b62bc83eda.png)

Reply:
Fixed!